wetterdienst/file_path_handling/file_list_creation.py

```diff
@@ -39,7 +39,7 @@ def create_file_list_for_dwd_server(
     time_resolution = TimeResolution(time_resolution)
     period_type = PeriodType(period_type)
 
-    file_index = create_file_index_for_dwd_server(
+    file_index = create_file_index_for_climate_observations(
```

Comment:
Keep in mind: Do not mix up refactoring with adding new features

Reply:
It's part of the added features as there also is another index creation now, so that the original name of the function would be confusing. Like one function called create_file_index_for_dwd_server and one function create_file_index_for_radolan would seem to me confusing as radolan obviously is part of the dwd server